agenta-web/src/components/Playground/ParamsForm/ParamsForm.tsx

```diff
@@ -109,20 +108,19 @@ const ParamsForm: React.FC<Props> = ({
                                             alt={param.name}
                                         />
                                     )}
-                                {isPlaygroundComponent ? (
```

Comment:
@bekossy any idea why did we have this before ?

Reply:
Yes @aakrem, The Input is also being used in the playground
The Input field in the playground to run prompts

Reply:
I mean why any idea why do we have :
```
{isPlaygroundComponent ? (
```
how this deletion will affect non-playground components ?

Reply:
@bekossy @aakrem any blockers here?

Reply:
@mmabrouk no blockers
@aakrem In the Human Evaluation the input is disabled but in the playground it is not
<img width="1068" alt="Screen Shot 2024-06-20 at 8 16 12 AM" src="https://github.com/Agenta-AI/agenta/assets/99529776/7bd42271-ab03-4e42-9a67-4a6a9d746e46">

<img width="229" alt="Screen Shot 2024-06-20 at 8 16 51 AM" src="https://github.com/Agenta-AI/agenta/assets/99529776/dc32b8b1-ec72-44d1-9537-c5fcd559a21c">

<img width="1476" alt="Screen Shot 2024-06-20 at 8 17 15 AM" src="https://github.com/Agenta-AI/agenta/assets/99529776/19747ea4-b1ec-4b1f-b1ad-539e86d79d2e">